airbyte-integrations/connectors/source-stock-ticker-api-cdk/source_stock_ticker_api_cdk/schemas/stock_prices.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema" : "http://json-schema.org/draft-07/schema#",
+  "type" : "object",
+  "properties" : {
+    "date" : {
+      "type" : "string"
```

Comment:
better add null value to avoid issues when value is none or empty. 

"type" : ["null", "string"]